object.go

```diff
@@ -34,9 +34,27 @@ func (c Color) MarshalText() ([]byte, error) {
 	return []byte(c), nil
 }
 
+type DatabaseMention struct {
+	ID ObjectID `json:"id"`
+}
+
+type PageMention struct {
+	ID ObjectID `json:"id"`
+}
+
+type Mention struct {
+	Type            ObjectType       `json:"type,omitempty"`
```

Comment:
This is not an `ObjectType`. I think, it'd be better to make a new type like `MentionType` and add all possible variations into `const.go`

Reply:
Done